action.yml

```diff
@@ -15,6 +15,9 @@ inputs:
       Target image(s) that will be published to the registries configured in the file ${HOME}/.docker/config.json
       Type: CSV
     required: true
+  tar-file:
+    description: 'Optional path to save the image tar file. If set, the image will be saved as a tar file at this path.'
```

Comment:
This input is not `required` which implies it is optional, therefore there is no need to begin the description with the word `Optional`.

Reply:
Ok